src/runtime/opengl.cpp

```diff
@@ -773,6 +772,36 @@ WEAK TextureInfo *find_texture(GLuint tex) {
     return texinfo;
 }
 
+// Allocate new TextureInfo, fill it in, add to global list.
+static TextureInfo *new_texture(GLuint tex, int32_t min[4], int32_t extent[4], bool halide_allocated) {
```

Comment:
Why static instead of weak? Is there a reason to have a separate copy of this per pipeline?